packages/@aws-cdk/aws-events/lib/custom-event-bus.ts

```diff
@@ -0,0 +1,37 @@
+import { Construct } from '@aws-cdk/core';
+import { BaseEventBus, IEventBus } from './base-event-bus';
+
+export interface ICustomEventBus extends IEventBus {}
+
+export interface CustomEventBusProps {
+  /**
+   * The name of the custom event bus you are creating.
+   */
+  readonly eventBusName: string;
+}
+
+/**
+ * Defines a Custom Event Bus
+ *
+ * @resource AWS::Events::EventBus
+ */
+export class CustomEventBus extends BaseEventBus implements ICustomEventBus {
```

Comment:
Not sure why this is not called `EventBus`, and is implemented using the constructor that is currently on `BaseEventBus`?